src/envoy/utils/config.cc

```diff
@@ -28,10 +28,31 @@ namespace Utils {
 namespace {
 
 const std::string kV2Config("v2");
+const std::string kV1Config("v1");
 
 // The name for the mixer server cluster.
 const std::string kDefaultMixerClusterName("mixer_server");
 
+// ReadConfig() finds config from |json| that matches version |config_version|,
+// and parses config into |message|. Returns true if config is read and parsed
+// successfully.
+bool ReadConfig(const Json::Object &json, Message *message,
+                const std::string &config_version) {
```

Comment:
per style guide,  output argument should be the last one.

Reply:
Thanks for pointing that out. Fixed.